docker/docsite/Dockerfile

```diff
@@ -19,8 +19,11 @@ ARG USER
 ARG UID
 ARG GID
 
-RUN apt update -qqy \
-    && apt install -qqy --no-install-recommends \
+ENV DEBIAN_FRONTEND=noninteractive
```

Comment:
This change should not be needed, because it layers on top of `cobalt-build-base` ( see below ) in where you already set those changes. Same comments here and below.